OpenRA.Mods.Common/Widgets/Logic/Editor/MapEditorTabsLogic.cs

```diff
@@ -63,12 +63,6 @@ void SetupTab(string buttonId, string tabId, MenuType tabType)
 
 		MenuType SelectTab(MenuType newMenuType)
 		{
-			if (menuType == MenuType.Select)
-			{
-				editor.SetBrush(editor.DefaultBrush);
-				editor.DefaultBrush.ClearSelection();
-			}
-
```

Comment:
perhaps a selection tab could be added that allows you to go back to it without having to reselect the area 🤔